### PR TITLE
Travis-ci: added support for ppc64le build along with amd64

### PR DESCRIPTION
**Pull request by sanjaymsh**
Hi,
I have added support for ppc64le build on travis-ci in the branch . The travis-ci build log can be tracked on the link :https://travis-ci.com/github/sanjay-cpu/libva/builds/186059891 . I believe it is ready for the final review and merge.
Please have a look on it and if everything looks fine for you then please approve it for merge.

Thanks !!

### CODE REVIEW

**Comment by sanjaymsh** [comment]
Hi, Can someone have a look on this.

Thanks !!

**Comment by XinfengZhang** [comment]
hi @sanjaymsh , we already switch it to github actions, could you help to file a similar PR for github actions?

**Comment by sanjaymsh** [comment]
Hi @XinfengZhang As of now, there is no support for ppc64 on GH, team is working on that. So raising a PR ,not going to help us for some couple of months.

Thanks!!

**Review by dvrogozh — CHANGES_REQUESTED** [review]
Travis ci was disabled for the reason that it worked enormously slow and we don't want to return to this situation again when a change was not validated for days. Imho, in such a way that's not a ci. So, we will stay on github actions. As such, please, look into options to add corresponding support there. Considering your note that it is not yet available out of the box, I would consider:
1. Request missing support from github
2. Look into cross-compilation which can be added into current CI
2. Look into VMs, i.e. in a similar way freebsd is not available out of the box on github, but yet we have such CI coverage thru VM support. See https://github.com/intel/libva/blob/master/.github/workflows/freebsd.yml

Basically, choosing between return of travis ci and manual reaction on ppc64le build failures I really strongly prefer the latter.
